pypose/__init__.py

```diff
@@ -14,3 +14,24 @@
 from .lietensor.function import *
 from . import module, optim
 from .basics import *
+
+
+def digit_version(version_str):
+    digits = []
+    for x in version_str.split('.'):
+        if x.isdigit():
+            digits.append(int(x))
+        elif x.find('rc') != -1:
+            patch_version = x.split('rc')
+            digits.append(int(patch_version[0]) - 1)
+            digits.append(int(patch_version[1]))
+    return digits
+
+
+torch_min_ver = '2.0'
```

Comment:
this will need change once torch version is upgraded, better to read `requirements/runtime.txt`

Reply:
`requirements/runtime.txt` is outside the package, so it is hard to do an i/o read from it during runtime.